site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -129,348 +122,234 @@ If your extension needs to run on `file://` URLs or needs to operate in incognit
+
+`"enterprise.platformKeys"`
+: Gives access to the <a href="/docs/extensions/reference/enterprise_platformKeys/">chrome.enterprise.platformKeys</a> API.
+
+`"fileBrowserHandler"`
+: Gives access to the <a href="/docs/extensions/reference/fileBrowserHandler/">chrome.fileBrowserHandler</a> API.
+
+`"fileSystemProvider"`
+: Gives access to the <a href="/docs/extensions/reference/fileSystemProvider/">chrome.fileSystemProvider</a> API.
+
+`"fontSettings"`
+: Gives access to the <a href="/docs/extensions/reference/fontSettings/">chrome.fontSettings</a> API.
+
+`"gcm"`
+: Gives access to the <a href="/docs/extensions/reference/gcm/"><code>chrome.gcm</code></a> and <a href="https://developer.chrome.com/docs/extensions/reference/instanceID/"><code>chrome.instanceID</code></a> APIs.
+
+`"geolocation"`
+:Allows the extension to use the <a
+          href="https://developer.mozilla.org/docs/Web/API/Geolocation_API">geolocation API</a> without prompting the user for permission.
+
+`"history"`
+:Gives access to the <a href="/docs/extensions/reference/history/">chrome.history</a> API.
+
+`"identity"`
+: Gives access to the <a href="/docs/extensions/reference/identity/">chrome.identity</a> API.
+
+`"idle"`
+: Gives access to the <a href="/docs/extensions/reference/idle/">chrome.idle</a> API.
+
+`"loginState"`
+: Gives access to the <a href="/docs/extensions/reference/loginState/">chrome.loginState</a> API.
+
+`"management"`
+: Gives access to the <a href="/docs/extensions/reference/management/">chrome.management</a> API.
+
+`"nativeMessaging"`
+: Gives access to the <a href="/docs/extensions/mv3/nativeMessaging/">native messaging API</a>.
+
+`"notifications"`
+: Gives access to the <a href="/docs/extensions/reference/notifications/">chrome.notifications</a> API.
+
+`"offscreen"`
+: Gives access to the <a href="/docs/extensions/reference/offscreen/"><code>chrome.offscreen</code></a> API. 
+
+`"pageCapture"`
+: Gives access to the <a href="/docs/extensions/reference/pageCapture/">chrome.pageCapture</a> API.
+
+`"platformKeys"`
+: Gives access to the <a href="/docs/extensions/reference/platformKeys/">chrome.platformKeys</a> API.
+
+`"power"`
+: Gives access to the <a href="/docs/extensions/reference/power/">chrome.power</a> API.
+
+`"printerProvider"`
+:Gives access to the <a href="/docs/extensions/reference/printerProvider/">chrome.printerProvider</a> API.
+
+`"printing"`
+: Gives access to the <a href="/docs/extensions/reference/printing/">chrome.printing</a> API.
+
+`"printingMetrics"`
+: Gives access to the <a href="/docs/extensions/reference/printingMetrics/">chrome.printingMetrics</a> API.
+
+`"privacy"`
+: Gives access to the <a href="/docs/extensions/reference/privacy/">chrome.privacy</a> API.
+
+`"processes"`
+: Gives access to the <a href="/docs/extensions/reference/processes/">chrome.processes</a> API.
+
+`"proxy"`
+: Gives access to the <a href="/docs/extensions/reference/proxy/">chrome.proxy</a> API.
+
```

Comment:
```suggestion

`"runtime"`
: Gives access to the <a href="/docs/extensions/reference/runtime/">runtime</a> API.

```

Reply:
The runtime API doesn't require any permissions, unless you are using native messaging

https://developer.chrome.com/docs/extensions/reference/runtime/#perms